### PR TITLE
Lm01/solution

### PR DESCRIPTION
**Pull request by gaand**

### CODE REVIEW

File: lib/diagnostic.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const min = function (previous, current) {
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let total = array.reduce(sum);`
 const sum = function (previous, current) {
-
+  return previousValue + currentValue
 }
```
Comment (payne-chris-r) [inline lib/diagnostic.js]:
```diff
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let minimum = array.reduce(min);`
 const min = function (previous, current) {
-  return previousValue < currentValue ? previousValue : currentValue
+  return previous < current ? previous : current
 }

 // Question 2
@@ -15,7 +15,7 @@ const min = function (previous, current) {
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let total = array.reduce(sum);`
 const sum = function (previous, current) {
-  return previousValue + currentValue
+  return previous + current
 }
```

Comment (payne-chris-r) [inline lib/diagnostic.js]: Not sure what order these happened in, but my guess is you got it working and then rebased off of master? I'm not sure. What I DO know is that `previousValue` isn't defined unless we change the parameters to `previousValue` as well. On `master`, these are written as `...function (previous, current)`

**Review by payne-chris-r — COMMENTED** [review]
Tests don't pass

File: lib/diagnostic.js (hunk 1 above)
Comment (gaand) [inline lib/diagnostic.js]: I'm not sure what went wrong.  I'm pushing a correction.

**Pull request merged**